docker/lr-cartographer/Dockerfile

```diff
@@ -0,0 +1,153 @@
+# Start with a good base python3 image:
```

Comment:
I have a strong opinion here: 
this is a completely new docker system.
It has many non-trivial programs in it.
Therefore it does not belong to this repo.
It deserves its own repo.

Reply:
I've added issue #350 to track this.